.github/workflows/clone_with_pinned_version.yml

```
@@ -0,0 +1,49 @@
+name: JEDI Bundle Pinned Version Clone Test
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+
+jobs:
+  buildbundle:
+    name: Clone oops bundle
+    runs-on: ubuntu-latest
+
+    steps:
+
+    # Setup Python
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
```

Per the [docs](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python), updating `setup-python` action version to latest (`v5`).

```suggestion
      uses: actions/setup-python@v5
```